ld/README

```diff
@@ -27,17 +28,15 @@ specified, because the device specific features is usually dependent on the
 last characters of device name string. Note that the device name string search
 is case insensitive.
 
-DEVICE=stm32f407vgt6
+DEVICE=stm32f103rbt6
 
-Then, the user includes the file /ld/Makefile.linker exactly after the first
-target (usually all) has been defined.
+Then, the user includes the file /mk/genlink-*.mk into his Makefile (refer to
+/mk/README). The generated linker script file will contain sections rom and
+ram with appropriate initialization code, specified in linker file source
+linker.ld.S
```

Comment:
If there's already a readme in mk/README, shouldn't the readme here just talk about how to edit the devices.data file?